src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -84,10 +91,22 @@ export function ProfileSubpageHeader({
           },
         ],
         index: 0,
-        thumbDims: null,
       })
+    },
+    [openLightbox],
+  )
+
+  const onPressAvi = React.useCallback(() => {
+    if (
+      avatar // TODO && !(view.moderation.avatar.blur && view.moderation.avatar.noOverride)
```

Comment:
probably can remove this todo?

Reply:
don't have context on it so don't wanna touch it. can do but separate PR?